docs/01_tutorials/01_concepts.rst

```diff
@@ -68,7 +68,7 @@ The current implementation of Tianshou typically use the following reserved keys
 * ``info`` the info of step :math:`t` (in ``gym.Env``, the ``env.step()`` function returns 4 arguments, and the last one is ``info``);
 * ``policy`` the data computed by policy in step :math:`t`;
 
-When adding data to a replay buffer, the done flag will be inferred automatically from ``terminated``and ``truncated``.
+When adding data to a replay buffer, the done flag will be inferred automatically from ``terminated or truncated``.
```

Comment:
```suggestion
When adding data to a replay buffer, the done flag will be inferred automatically from ``terminated`` or ``truncated``.
```

Reply:
just a note: there is another ``terminated or truncated`` line [a few lines above](https://github.com/strakam/tianshou/blob/35c4cee505e187cd059c329412377b1b6a87b33f/docs/01_tutorials/01_concepts.rst?plain=1#L66), but I dont want to bother you with another PR .